pkg/controller/servicemesh/webhooks/mutation/controlplane.go

```diff
@@ -237,3 +256,23 @@ func (m *smcpv2mutator) OldVersion() string {
 func (m *smcpv2mutator) GetProfiles() []string {
 	return m.smcp.Spec.Profiles
 }
+
+func (m *smcpv2mutator) NewOpenShiftRoute() *bool {
```

Comment:
I think `func IsOpenShiftRouteEnabled() bool` would make more sense

Reply:
There are new and old SMCP in the mutator. I tried to follow the naming of other getter function to reflect getting value from the new SMCP.